lib/rules/no-tests-contain-correct.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+  meta: {
+    docs: {
+      description:
+        'Prevent the usage of the word "correct" in test statements.',
+      category: 'Best Practices',
+      recommended: false,
+      uri:
+        'https://github.com/Shopify/eslint-plugin-shopify/blob/master/docs/rules/no-tests-contain-correct.md',
+    },
+  },
+  create(context) {
+    const ignore = (context.options[0] && context.options[0].ignore) || [];
+    const ignoredFunctionNames = ignore.reduce((accumulator, value) => {
+      accumulator[value] = true;
+      return accumulator;
+    }, Object.create(null));
+
+    function isIgnoredFunctionName(node) {
+      return ignoredFunctionNames[node.callee.name];
+    }
+
+    return {
+      CallExpression(node) {
+        const erroneousMethod = descriptionContainsCorrect(node);
+
+        if (erroneousMethod && !isIgnoredFunctionName(node)) {
+          context.report({
+            message: `{{ method }} statement should not contain the word "correct"`,
+            data: {method: erroneousMethod},
+            node,
+          });
+        }
+      },
+    };
+  },
+};
+
+function isItTestOrDescribeFunction(node) {
```

Comment:
Probably good to catch the skipped versions (`fit`/ `it.only` and the same for the other kinds)